doozerlib/olm/bundle.py

```diff
@@ -24,6 +24,24 @@ class OLMBundle(object):
     def __init__(self, runtime):
         self.runtime = runtime
 
+    def find_bundle_for(self, operator_nvr):
+        """Check if a bundle already exists for a given `operator_nvr`.
+
+        :param string operator_nvr: Operator NVR (format: my-operator-v4.2.30-202004200449)
+        :return string: NVR of latest found bundle build, or None.
+        """
+        self.operator_nvr = operator_nvr
+        self.get_operator_buildinfo()
+
+        _rc, out, _err = exectools.cmd_gather(
+            'brew list-builds --quiet --package={}'.format(self.bundle_brew_component)
```

Comment:
this will list every build since the beginning of time - i think we can safely restrict it to those which have been tagged for the release candidate branch? also we have `brew.get_tagged_builds()` for this which ought to be faster than shelling out.

Reply:
much better!! done: https://github.com/openshift/doozer/pull/314/commits/50e050b7e9d6e7747e9a56ac25b4500772c7cb9b